spec/amp-story-player.md

```diff
@@ -0,0 +1,201 @@
+  </head>
+  <body>
+    <h1>This is a NON-AMP page that embeds a story below:</h1>
+    <amp-story-player style="width: 360px; height: 600px;">
+      <a
+        href="https://example.com/story.html"
+        style="--story-player-poster: url('https://example.com/img/promo3x4.jpg');"
+      >
+        <span>A description for my story.</span>
+      </a>
+    </amp-story-player>
+  </body>
+</html>
+```
+
+The high-level API for the player looks like the following:
+
+[example preview="top-frame" playground="true"]
+
+```html
+<head>
+  <script
+    async
+    src="https://cdn.ampproject.org/amp-story-player-v0.js"
+  ></script>
+  <link
+    href="https://cdn.ampproject.org/amp-story-player-v0.css"
+    rel="stylesheet"
+    type="text/css"
+  />
+  <style injected>
+    .story-player-iframe {
+      background-size: cover;
+      display: block;
+      position: relative;
+      height: 100%;
+      width: 100%;
+      flex: 0 0 100%;
+    }
+
+    .title {
+      background: linear-gradient(
+        to bottom,
+        rgba(0, 0, 0, 0),
+        rgba(0, 0, 0, 1)
+      );
+      display: block;
+      position: absolute;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      color: #fff;
+      font-size: 40px;
+      padding: 88px 32px 32px 32px;
+      line-height: 1.25;
+      text-shadow: 3px 3px #000;
+    }
+  </style>
+</head>
+<body>
+  <h1>This is a NON-AMP page that embeds a story below:</h1>
+  <amp-story-player style="width: 360px; height: 600px;">
+    <a
+      href="https://www-washingtonpost-com.cdn.ampproject.org/v/s/www.washingtonpost.com/graphics/2019/lifestyle/travel/amp-stories/a-locals-guide-to-what-to-eat-and-do-in-mexico-city/"
+      data-poster-portrait-src="https://www-washingtonpost-com.cdn.ampproject.org/v/s/www.washingtonpost.com/graphics/2019/lifestyle/travel/amp-stories/a-locals-guide-to-what-to-eat-and-do-in-mexico-city/img/promo3x4.jpg"
+      class="story"
+    >
+      <span class="title"
+        >A local’s guide to what to eat and do in Mexico City</span
+      >
+    </a>
+  </amp-story-player>
+</body>
+```
+
+[/example]
+
+### Attributes
+
+The inline width and height ensures that the player will not cause any jumps in your website while the script is being loaded. Feel free to modify the values, but we recommend maintaining a 3:5 aspect ratio.
+
+<table>
+  <tr>
+    <td width="40%"><strong>width [recommended]</strong></td>
+    <td>Width of the player in pixels.</td>
+  </tr>
+  <tr>
+    <td width="50%"><strong>height [recommended]</strong></td>
+    <td>Height of the player in pixels.</td>
+  </tr>
+</table>
+
+## Specify embedded story
+
+The `<amp-story-player>` component contains one or more `<a>` tags, each of which represent a story. Use the `href` attribute to point to a particular story.
+
+To provide a better experience for the user while the story is being loaded, as well as allowing search engines to crawl the embedded stories, you can provide some text inside the anchor tags.
```

Comment:
I'd update to:
Place descriptive text of the story within the `<a>` tag. This provides a better user experience and allows search engines to crawl embedded stories.